app/lib/dfe/reference_data/degrees.rb

```diff
@@ -1,730 +1,669 @@
+          '8b5bb841-c7fd-43e3-87d8-dc9653c54dda' => { 'name' => 'foundation degree', synonyms => [], level => '5' },
+          '2e4d3d83-bbf7-4d37-b451-0f6bbf3ac32a' => { 'name' => 'higher national diploma', synonyms => ['HND'], level => '5' },
+          'ab446a38-71d5-45c2-8ad6-23990dc66f16' => { 'name' => 'level 5 award', synonyms => [], level => '5' },
+          '679215a9-6b4f-41b2-b7d9-f179c3f241ed' => { 'name' => 'level 5 certificate', synonyms => [], level => '5' },
+          '2810949c-b17f-4b08-8461-f4e3775e81f3' => { 'name' => 'level 5 diploma', synonyms => [], level => '5' },
+          'fdd44a44-ce01-4a4c-bc99-1e17e69ad8fe' => { 'name' => 'level 5 national vocational qualification', synonyms => ['NVQ'], level => '5' },
+          '77c6065f-de57-4094-b588-c1317b0f0b3e' => { 'name' => 'degree apprenticeship', synonyms => [], level => '6' },
+          'b580a760-da23-4d38-b803-62ae11de6a65' => { 'name' => 'ordinary degree', synonyms => ['bachelors degree', "bachelor's degree"], level => '6' },
+          'a4a57564-1707-4aed-8231-1be922f1acee' => { 'name' => 'graduate certificate', synonyms => [], level => '6' },
+          'fc519e23-9c57-4a10-bba7-ea609b11e20f' => { 'name' => 'graduate diploma', synonyms => [], level => '6' },
+          '8c19f663-73fb-4875-b705-766ebd2726ed' => { 'name' => 'level 6 award', synonyms => [], level => '6' },
+          'b5d7fe12-c2e9-4e35-82ed-bf18231ad594' => { 'name' => 'level 6 certificate', synonyms => [], level => '6' },
+          '4290f584-fb7d-420e-8a21-1d6ef50e2876' => { 'name' => 'level 6 diploma', synonyms => [], level => '6' },
+          'c0ada43b-9b18-424a-9ddd-47af929d4220' => { 'name' => 'level 6 NVQ', synonyms => [], level => '6' },
+          '4b7f4349-a981-4441-8f4d-ad2d0e57c8e9' => { 'name' => "master's degree", synonyms => ['masters degree'], level => '7' },
+          '67c8cb38-5ecc-4bd6-b2fe-847348c8cde7' => { 'name' => 'level 7 award', synonyms => [], level => '7' },
+          '25e0b680-c960-48ec-bd4a-338c8bc9c116' => { 'name' => 'level 7 certificate', synonyms => [], level => '7' },
+          '01d73c15-fefd-4a5b-a211-e310eab1cd93' => { 'name' => 'level 7 diploma', synonyms => [], level => '7' },
+          '61f9f333-c77e-4a93-8837-ce86d65ecfad' => { 'name' => 'level 7 national vocational qualification', synonyms => ['NVQ'], level => '7' },
+          '2bda548c-38f4-428a-94a9-ce248b6d582b' => { 'name' => 'postgraduate certificate', synonyms => [], level => '7' },
+          'de63513e-edaf-4e54-ad4a-803fe0ee9ec6' => { 'name' => 'postgraduate certificate in education', synonyms => ['PGCE'], level => '7' },
+          'bda17e7c-aca7-488a-a7c0-9420eb8b7204' => { 'name' => 'postgraduate diploma', synonyms => [], level => '7' },
+          '181eb0ae-eeb8-487b-9c6a-a4ee3e1907b6' => { 'name' => 'doctorate, for example doctor of philosophy', synonyms => ['PhD'], level => '8' },
```

Comment:
Move the 'For example' into a `hint` attribute, so that we can replay the name without the description in some contexts?

```suggestion
          '181eb0ae-eeb8-487b-9c6a-a4ee3e1907b6' => { 'name' => 'doctorate', 'hint' => 'for example doctor of philosophy', synonyms => ['PhD'], level => '8' },
```

Reply:
Done